yourgov/components/FormMobileFoodVendorPermit/task1/FormTask1Step5.tsx

```diff
@@ -0,0 +1,148 @@
+import { Controller, SubmitHandler, useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { DateRangePicker } from '@ag.ds-next/react/date-range-picker';
+import { FormStack } from '@ag.ds-next/react/form-stack';
+import { GroupedFields } from '@ag.ds-next/react/grouped-fields';
+import { Stack } from '@ag.ds-next/react/stack';
+import { TimeInput } from '@ag.ds-next/react/time-input';
+import { DeepPartial } from '../../../lib/types';
+import { FormPageAlert } from '../FormPageAlert';
+import { hasMultipleErrors, parseDateField } from '../utils';
+import { type ShallowErrors } from '../FormState';
+import { StepActions } from '../StepActions';
+import { useGlobalForm } from '../GlobalFormProvider';
+import { FormTask1Container } from './FormTask1Container';
+import { useFormTask1Context } from './FormTask1Provider';
+import {
+	task1Step5FormSchema,
+	type Task1Step5FormSchema,
+} from './FormTask1FormState';
+
+function transformDefaultValues(step?: DeepPartial<Task1Step5FormSchema>) {
+	const from = step?.tradingPeriod?.from as string | Date | undefined;
+	const to = step?.tradingPeriod?.from as string | Date | undefined;
```

Comment:
Similar change based on update to `DeepPartial`:

```suggestion
	const from = step?.tradingPeriod?.from;
	const to = step?.tradingPeriod?.from;
```